gammapy/estimators/points/tests/test_core.py

```diff
@@ -336,3 +336,16 @@ def test_flux_points_plot_no_error_bar():
     flux_points = FluxPoints.from_table(table)
     with mpl_plot_check():
         _ = flux_points.plot(sed_type="dnde")
+
+
+def test_fp_no_is_ul():
```

Comment:
```suggestion
@requires_data()
def test_fp_no_is_ul():
```

The decorator is required for CI to run.